Osiris/Hacks/Visuals.cpp

```diff
@@ -393,3 +393,22 @@ void Visuals::skybox() noexcept
         memory->loadSky(sv_skyname->string);
     }
 }
+
+void Visuals::showVelocity() noexcept
+{
+    if (!config->visuals.showvelocity.enabled || !localPlayer->isAlive())
+        return;
+
+    float velocity = localPlayer->velocity().length2D();
+    std::wstring velocitywstr{ L"Velocity: " + std::to_wstring(static_cast<int>(velocity)) };
+
+    interfaces->surface->setTextFont(Surface::font);
+    if (config->visuals.showvelocity.rainbow)
+        interfaces->surface->setTextColor(rainbowColor(memory->globalVars->realtime, config->visuals.showvelocity.rainbowSpeed));
```

Comment:
` interfaces->surface->setTextColor(rainbowColor(config->visuals.showvelocity.rainbowSpeed));`
Should fix the compilation error

Reply:
Ok i will fix it but i swear it compiled fine for me